imports/plugins/core/orders/client/components/invoiceActions.js

```diff
@@ -33,7 +46,7 @@ class InvoiceActions extends Component {
     return (
       <div className="invoice-summary">
         <span className="invoice-label adjusted-total">
-          <strong><Translation defaultValue="Adjusted Total" i18nKey="admin.invoice.adjustedTotal"/></strong>
+          <strong><Translation defaultValue="Adjusted Total" data-i18n="admin.invoice.adjustedTotal"/></strong>
```

Comment:
The `Translation` component has `i18nKey` as a prop, not sure of `data-i18n` as a prop it receives.